src/main/java/use_cases/GameCreationRequestModel.java

```diff
@@ -0,0 +1,39 @@
+package use_cases;
+
+import java.util.HashMap;
+
+/**
+ * A request model containing User provided information on the Players to be made in a Game.
+ */
+public class GameCreationRequestModel {
+    private HashMap<String, Boolean> playersInfo;
```

Comment:
Consider turning this HashMap into a class since this hash map and the one in GameCreationController are essentially the same. Turning it into a class can also make the code extensible for when we want to add more properties to a player.

Reply:
The hashmap in the controller is used in constructing the response model. I do see your point on making this a class and more extensible though, and will keep it in mind going forward.